.github/dependabot.yml

```diff
@@ -1,6 +1,19 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
+  - package-ecosystem: "bundler"
     directory: "/"
+    # Required to fetch dependencies from our private gem server,
+    # see https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#insecure-external-code-execution.
+    # This is safe as we are fetching gems from our own private registry.
+    insecure-external-code-execution: allow
+    # "open-pull-requests-limit" only applies to auto-updates, and not security
+    # updates. A limit of 0, ensures that dependabot only raises PRs for security
+    # updates.
+    # ref: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit:
+    open-pull-requests-limit: 0
+    registries:
+      - "ruby-github"
```

Comment:
I think we also need to configure this in this file similar to what we have here?: https://github.com/thrivadev/platform-conductor/blob/master/.github/dependabot.yml#L2-L6

Reply:
That's only if we are using private gems (hosted by our private gem repository), which at the moment we're not so I think we can leave it out.

Reply:
Ah of course, in that case I think we need to remove the `registries` field in the `updates` blob as that tells it what registries it can use. Since we haven't defined any top-level registries then it'll fail to recognise `ruby-github` right?

See: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#registries

Reply:
Removed!